site/gatsby-site/migrations/2022.09.19T20.06.43.is-incident.js

```diff
@@ -0,0 +1,14 @@
+const config = require('../config');
+
+/**
+ *
+ * @param {{context: {client: import('mongodb').MongoClient}}} context
+ */
+
+exports.up = async ({ context: { client } }) => {
```

Comment:
Does it make sense to have an `export.down` in this case?

Reply:
I think it does not because it is not used by anything anyway

Reply:
OK, that makes sense to me too.